Decred Wallet/Features/Transactions/TransactionsViewController.swift

```diff
@@ -9,54 +9,57 @@
 import UIKit
 import Dcrlibwallet
 
-class TransactionHistoryViewController: UIViewController {
+class TransactionsViewController: UIViewController {
+    @IBOutlet weak var headerTopConstraint: NSLayoutConstraint!
+    @IBOutlet weak var headerStackView: UIStackView!
+    @IBOutlet weak var headerBottomConstraint: NSLayoutConstraint!
     @IBOutlet weak var syncInProgressLabel: UILabel!
+    @IBOutlet var transactionsTableView: UITableView!
+    @IBOutlet var transactionFilterDropDown: DropMenuButton!
+    @IBOutlet var transactionSorterDropDown: DropMenuButton!
```

Comment:
```suggestion
    @IBOutlet var txSortOrderDropDown: DropMenuButton!
```
- Finally came to me, "sorters" should be "sort orders", rename appropriately.
- To shorten variable names, replace `transaction` with `tx` where possible.